userspace/libsinsp/examples/test.cpp

```diff
@@ -20,12 +20,26 @@ limitations under the License.
 #include <signal.h>
 #include <unistd.h>
 #include <sinsp.h>
+#include <functional>
 #include "util.h"
 
 using namespace std;
 
 static bool g_interrupted;
-static const uint8_t g_backoff_timeout_secs = 2; 
+static const uint8_t g_backoff_timeout_secs = 2;
+
+sinsp_evt* get_event(sinsp& inspector);
+
+#define PROCESS_DEFAULTS "*%evt.num %evt.time %evt.category %container.id %proc.ppid %proc.pid %evt.type %proc.exe %proc.cmdline %evt.args"
+
+// Formatters used with JSON output
+sinsp_evt_formatter* default_formatter = nullptr;
```

Comment:
Can we make these `static` ? :D 
I am sorry to bother you!

Reply:
It's no bother at all. Would it be better to make them part of a nameless namespace? I believe that is the more "C++" way of doing it.

Reply:
Given that it is only used in this small example, i won't deal with namespaces; feel free to do that though!